test/speed-index.js

```diff
@@ -88,7 +112,16 @@ test('speed indexes calculated for 3 frame (blank @1s, content @2s, more content
 	speedIndex.calculatePerceptualProgress(data.frames);
 	const indexes = speedIndex.calculateSpeedIndexes(data.frames, data);
 	t.is(Math.floor(indexes.speedIndex), 2040);
-	t.is(Math.floor(indexes.perceptualSpeedIndex), 2066);
+	t.is(Math.floor(indexes.perceptualSpeedIndex), 2030);
+});
+
+test('speed indexes calculated for realistic trace', async t => {
+	const data = await frame.extractFramesFromTimeline('./assets/progressive-app-updated.json');
```

Comment:
yeah good call, I realized "updated" makes no sense outside of this commit haha, done